content/Consumer Privacy Regulations CCPA and GDPR - Pablo Blanco.md

```diff
@@ -0,0 +1,240 @@
+# Consumer Privacy Regulations CCPA and GDPR
```

Comment:
Let's start to think more about search trends. 
"CCPA compliance" has a huge growth in searches (https://trends.google.com/trends/explore?geo=US&q=GDPR%20CCPA) . "GDPR for dummies" also has many searches. 

Maybe we can name this: GDPR and CCPA Compliance for Dummies

Reply:
I don't like the "Dummies" part of the title, but is definitely a better option than the original one. Let's see @andrea-fajardo thoughts about this one.